package.json

```diff
@@ -17,6 +17,8 @@
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.57",
+    "@nebula.gl/edit-modes": "^1.0.3",
```

Comment:
Added as a dev dependency to be able to add the `FeatureSelectionLayer.test.js` test